src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -84,11 +83,24 @@ trait BackendCall {
    * The Workflow and Call to invoke.  It is assumed that in the creation
    * of a BackendCall object that the 'call' would be within the workflow
    */
-  def workflowDescriptor: WorkflowDescriptor
-  def key: BackendCallKey
+  def callDescriptor: BackendCallDescriptor
+  def workflowDescriptor = callDescriptor.workflowDescriptor
+  def key = callDescriptor.key
   def call = key.scope
 
   /**
+    * Inputs to the call.  For example, if a call's task specifies a command like this:
+    *
+    * File some_dir
+    * command {
+    *   ls ${some_dir}
+    * }
+    *
+    * Then locallyQualifiedInputs could be Map("some_dir" -> WdlFile("/some/path"))
+    */
+  def locallyQualifiedInputs: CallInputs = callDescriptor.locallyQualifiedInputs
+
+  /**
    * Backend which will be used to execute the Call
    */
   def backend: Backend
```

Comment:
Perhaps, this can also be derived from the `CallDescriptor.workflowDescriptor` ?